sources/packages/backend/libs/src-sql/sfas-restrictions/Bulk-insert-sfas-mapped-restrictions.sql

```diff
@@ -22,6 +22,7 @@ FROM
         WHEN sfas_restrictions.code = 'A12' THEN '12'
         WHEN sfas_restrictions.code = 'AF1' THEN 'AF'
         WHEN sfas_restrictions.code = 'B2D' THEN 'B2'
+        WHEN sfas_restrictions.code = 'SINR' THEN 'LGCY'
```

Comment:
I do not believe that SINR will be sent from SFAS and I do not see a technical AC to add this condition.

Please let me know if I am missing something here?

Reply:
Please check `Bulk-insert-legacy-restrictions.sql`. This query will update with a `LGCY` if a restriction from SFAS is not found in SIMS.